Makefile

```diff
@@ -0,0 +1,37 @@
+# Makefile for eoapi-k8s
+
+# Variables
+HELM_REPO_URL=https://devseed.com/eoapi-k8s/
+HELM_CHART_NAME=eoapi/eoapi
+PGO_CHART_VERSION=5.5.2
+
+.PHONY: all deploy minikube help
+
+# Default target
+all: deploy
+
+deploy:
+	@echo "Installing dependencies."
+	@command -v helm >/dev/null 2>&1 || { echo "helm is required but not installed"; exit 1; }
+	helm install --set disable_check_for_upgrades=true pgo oci://registry.developers.crunchydata.com/crunchydata/pgo --version $(PGO_CHART_VERSION)
+	@echo "Adding eoAPI helm repository."
+	@helm repo add eoapi $(HELM_REPO_URL)
+	@echo "Installing eoAPI helm chart."
+	@cd ./helm-chart && \
+	helm install --namespace eoapi --create-namespace --set gitSha=$$(git rev-parse HEAD | cut -c1-10) eoapi ./eoapi
+
+minikube:
+	@echo "Starting minikube."
+	@command -v minikube >/dev/null 2>&1 || { echo "minikube is required but not installed"; exit 1; }
+	minikube start
+	# Deploy eoAPI via the regular helm install routine
+	@make deploy
+	minikube addons enable ingress
+	@echo "eoAPI is now available at:"
+	@minikube service ingress-nginx-controller -n ingress-nginx --url | head -n 1
```

Comment:
```
eoAPI is now available at:
http://127.0.0.1:51275
❗  Because you are using a Docker driver on darwin, the terminal needs to be open to run it.
```

This is what I get outputted at the end (if I apply the build fix).

The url resolves fine, but the make command doesn't complete, I just get the last line and the process is taking up my terminal instance...

Reply:
Ah, seems it needs to maintain the terminal session to hold open the tunnel. I think that's okay then.

Reply:
Cool! Thanks for testing it.